terraform/modules/networking/firewall.tf

```diff
@@ -185,13 +185,37 @@ resource "aws_networkfirewall_rule_group" "stateless_main" {
   }
 }
 
+locals {
+  subnet_firewall_rules = [
+    for name, config in local.firewall_config : join("\n", ["# ${name}", join("\n", concat(
```

Comment:
This is definitely a little complicated to follow, but don't think I've got any helpful suggestions on what we can actually do about it